kubernetes/kubernetes/src/main/java/org/arquillian/cube/kubernetes/impl/KubernetesAssistant.java

```diff
@@ -219,7 +226,13 @@ public void deployAll(String applicationName, Path directory) throws IOException
         }
     }
 
-    private void deploy(InputStream inputStream) throws IOException {
+    /**
+     * Deploys application reading resources from specified InputStream
+     *
+     * @param inputStream  where resources are read
+     * @throws IOException
+     */
+    public void deploy(InputStream inputStream) throws IOException {
```

Comment:
Do we really need to make it public?

Reply:
I think this gives just another means of deploying application, one example where I saw this could be used was for launcher-backend. So, I feel it makes sense to having it as public.